tests/testStreamOpen.cpp

```diff
@@ -437,6 +441,50 @@ TEST_F(StreamOpenInput, AAudioInputSetAttributionTag){
     }
 }
 
+TEST_F(StreamOpenInput, AAudioInputSetSpatializationBehavior) {
+    mBuilder.setDirection(Direction::Input);
+    mBuilder.setSpatializationBehavior(SpatializationBehavior::Auto);
+    ASSERT_TRUE(openStream());
+    ASSERT_EQ(mStream->getSpatializationBehavior(), SpatializationBehavior::Auto);
+    ASSERT_TRUE(closeStream());
+}
+
+TEST_F(StreamOpenOutput, AAudioOutputSetSpatializationBehavior) {
+    mBuilder.setDirection(Direction::Output);
+    mBuilder.setSpatializationBehavior(SpatializationBehavior::Never);
+    ASSERT_TRUE(openStream());
+    ASSERT_EQ(mStream->getSpatializationBehavior(), SpatializationBehavior::Never);
+    ASSERT_TRUE(closeStream());
+}
+
+TEST_F(StreamOpenOutput, AAudioOutputSetSpatializationBehaviorUnspecified) {
+    mBuilder.setDirection(Direction::Output);
+    mBuilder.setSpatializationBehavior(SpatializationBehavior::Unspecified);
```

Comment:
Add In/Out tests that does not call setSpatializationBehavior() to make sure we get the same Never.